### ISSUE TITLE
giving a description for arguments

### ISSUE THREAD

**Issue by noraj** [davetron5000/gli#332]
[DSL#arg] lacks a `:desc` option to describe the argument in order to specify what is expected.

For example, in other tools you often see description for arguments:

![haiti help message](https://github.com/user-attachments/assets/f33e3b0b-f259-499e-bba3-52d464059116)
![git add help message](https://github.com/user-attachments/assets/fa92656b-9319-459c-bb24-2382810b7288)

The temporary workaround I found was to describe them in the `long_desc`:

```ruby
        duzdu.desc 'TODO'
        duzdu.long_desc "<name>: DNS name, A record. The domain is automatically appended, e.g. test ➡️ test.example.org\n\n<ip>: IP address."
        duzdu.arg :'<name>'
        duzdu.arg :'<ip>'
        duzdu.command :add do |add|
          add.action do |_global_options, options, _args|
            duz = duzdu_init(options)
            duz.display('addv4')
          end
        end
```

hacking the arguments' description into the command long description:

![hacking the arguments' description into the command long description](https://github.com/user-attachments/assets/022cbf4a-6faa-4f4f-8769-4ab0004c2848)

**Comment by davetron5000** [comment]
Yeah, this seems useful.  Could be complicated to document and format but there's no philosophical reason why this isn't there.
